### PR TITLE
chore: add PNAM inventory items

### PR DESCRIPTION
**Pull request by jniles**
We've added the PNAM inventory items, groups, and types, and ensured that the database builds and integration tests pass.

This is the list it was based off: [MEG_Nomenclature Rationalisée PNAM.xlsx](https://github.com/IMA-WorldHealth/bhima/files/4421647/MEG_Nomenclature.Rationalisee.PNAM.xlsx)

### CODE REVIEW

**Comment by mbayopanda** [comment]
@jniles having this list is great, it seems like all tests are not succeed yet

**Comment by jniles** [comment]
Yeah, still fighting it.  I'll ping you when I need a review.

**Comment by jniles** [comment]
@mbayopanda this is finally ready for a review.

File: client/src/modules/inventory/inventory.service.js [hunk 1]
```diff
@@ -41,12 +43,31 @@ function InventoryService(
   };
 
   function downloadInventoriesTemplate() {
-    $http.get(INVENTORY_IMPORT_URL.concat('template_file'))
+    service.$http.get(INVENTORY_IMPORT_URL.concat('template_file'))
       .then(response => {
-        return util.download(response, 'Template Inventory', 'csv');
+        return service.util.download(response, 'Template Inventory', 'csv');
```
Comment (mbayopanda) [inline client/src/modules/inventory/inventory.service.js:48]: 👍

**Review by mbayopanda — APPROVED** [review]
LGTM

The hard part was to understand why you decide to HttpCache for this module, what is the purpose of using it here.

**Comment by jniles** [comment]
> 
> 
> LGTM
> 
> The hard part was to understand why you decide to HttpCache for this module, what is the purpose of using it here.

Good question.  The main reason is that I discovered that navigating from the SearchModal -> Inventory List sent 3 identical HTTP requests that each had to load 3K inventory items.  That made the E2E test time out over and over.  So, I put HTTPCache so that it only ever sent a single HTTP request to the server.

**Comment by mbayopanda** [comment]
@jniles could you fix the conflict

**Comment by jniles** [comment]
@mbayopanda, will do.  I'll ping you when it is done.

**Comment by jniles** [comment]
@mbayopanda fixed and tests pass.

**Pull request merged**